package.json

```diff
@@ -11,7 +11,8 @@
     "stylelint": "9.1.1",
     "stylelint-config-standard": "18.2.0",
     "webpack": "4.3.0",
```

Comment:
Since we're updating the dependencies anyway, let's update this one to the latest too?